lib/jnpr/junos/utils/sw.py

```diff
@@ -967,7 +972,7 @@ def reboot(self, in_min=0, at=None, all_re=True):
         else:
             cmd = E('request-reboot', E('at', str(at)))
 
-        if all_re is True:
+        if all_re is True :
```

Comment:
why extra space?

Reply:
That was an accident. Thanks for catching it. I have corrected it.